packages/workspace/src/common/test/mock-workspace-server.ts

```diff
@@ -22,4 +22,6 @@ export class MockWorkspaceServer implements WorkspaceServer {
     getRoot(): Promise<string | undefined> { return Promise.resolve(''); }
 
     setRoot(uri: string): Promise<void> { return Promise.resolve(); }
+
+    getRecentRoots(): Promise<string[]> { return Promise.resolve(['']); }
```

Comment:
Why not an empty array? Just asking though...

Reply:
ok, I was just initializing with an empty string, but just [] is fine too